backend/app/routes/admin/forgotPassword.js

```diff
@@ -12,16 +12,16 @@ module.exports = async (req, res, next) => {
       statusCode: 400,
       message: 'Invalid login credentials',
       user: email,
-      errStack: 'Auth Login',
+      errStack: 'User not found',
     });
     next(error);
     return false;
   }
 
   // Setting EMAIL as the token payload
   const JWTPayload = { email };
-
-  const token = await generateJWT(JWTPayload, '1h');
+  const JWT_EXPIRY_TIME = '1h';
```

Comment:
This constant should move to the constants file

Reply:
name should be JWT_EMAIL_EXPIRES_IN

Reply:
Done